tests/examples/Fervo_Norbeck_Latimer_2024.txt

```diff
@@ -3,49 +3,70 @@
 ---   "Commercial-Scale Demonstration of a First-of-a-Kind Enhanced Geothermal System."
 ---   https://doi.org/10.31223/X52X0B
 
-Has Nonvertical Section, True
-Multilaterals Cased, True
-Well Geometry Configuration, 4,       ----L
-Number of Multilateral Sections, 2,  ---Two parallel horizontal sections
-Well Drilling Cost Correlation,3,				--- [-] Use built-in well drilling cost correlation #3 = vertical large diameter
-Horizontal Well Drilling Cost Correlation,4,				--- [-] Use built-in well drilling cost correlation #4 = deviated large diameter
-
+***Reservoir Parameters***
 Reservoir Model, 1
+Reservoir Volume Option,1,					---Should be 1 2 3 or 4. See manual for details.
+Reservoir Density, 2700
 Reservoir Depth, 2347 meter
-Number of Segments, 1
-Gradient 1, 68
-Number of Production Wells, 1
-Number of Injection Wells, 1
-Production Well Diameter, 7
-Injection Well Diameter, 7
-Ramey Production Wellbore Model, 1
-Injection Wellbore Temperature Gain, 0
-Production Flow Rate per Well, 63
-
+Reservoir Heat Capacity,1000,					---[J/kg/K]
+Reservoir Thermal Conductivity, 2.7
+Number of Fractures, 100
 Fracture Shape, 4
 Fracture Height, 100
 Fracture Width, 991 meter
 Fracture Separation, 10 meter
+Number of Segments,1,						---[-]
+Gradient 1, 68
+Drawdown Parameter,.00002,					---[kg/s/m2]
+Maximum Temperature, 500
 
-Reservoir Volume Option, 1
-Number of Fractures, 100
-Water Loss Fraction, .02
-Productivity Index, 5
-Injectivity Index, 5
+*** Wellbore Parameters***
+Number of Injection Wells, 1,					---[-]
+Number of Production Wells,1,					---[-]
+Well Geometry Configuration, 4,       ----L
+Has Nonvertical Section, True
+Multilaterals Cased, True
+Number of Multilateral Sections, 2,  ---Two parallel horizontal sections
+Total Nonvertical Length, 990 meter
+Well Drilling Cost Correlation,3,				--- [-] Use built-in well drilling cost correlation #3 = vertical large diameter
+Horizontal Well Drilling Cost Correlation,4,				--- [-] Use built-in well drilling cost correlation #4 = deviated large diameter
+Production Flow Rate per Well, 63
+Production Well Diameter, 7
+Injection Well Diameter, 7
 Injection Temperature, 50
-Maximum Drawdown, 1
-Reservoir Heat Capacity, 1000
-Reservoir Density, 2700
-Reservoir Thermal Conductivity, 2.7
-End-Use Option, 1
-Economic Model, 1
-Power Plant Type, 4
-Circulation Pump Efficiency, .8
-Utilization Factor, .9
-Surface Temperature, 20
-Ambient Temperature, 20
+Injection Wellbore Temperature Gain,3,				---[deg.C]
+Injectivity Index, 5,						---[kg/s/bar]
+Productivity Index, 5
+Ramey Production Wellbore Model,1,				---Should be 0 (disabled) or 1 (enabled)
+Utilization Factor,.9,						---[-]Water Loss Fraction, .02x
+Water Loss Fraction,0.02,						---[-]
+Maximum Drawdown,1,						---[-] no redrilling considered
+
+***Surfaceplant Parameters***
+Ambient Temperature, 10.27,						---[deg.C]
+Surface Temperature, 10.27
```

Comment:
Are these values from the paper?

Reply:
From this website as the mean annual average:
https://weatherspark.com/y/2384/Average-Weather-in-Milford-Utah-United-States-Year-Round